README.md

```diff
@@ -313,3 +313,31 @@ This can by compiled using a command like the following after running
 `make install` on VkRunner:
 
     cc -o myrunner myrunner.c $(pkg-config --cflags --libs vkrunner)
+
+## Android
+
+VkRunner supports Android NDK build which generates the VkRunner static library
+for Android.
+
+- Download [Android NDK](https://developer.android.com/ndk/downloads/).
+- Run the following commands:
+
+```
+cd <vkrunner-dir>
+
+mkdir build && cd build
+cmake .. && make
```

Comment:
The generation of config.h determines whether the platform is big-endian or little-endian.  But the Android build is a cross-compile.  The answer you get from CMake is for the host platform, and so does not necessarily match the answer for the cross compile.

For the android case, you should probably use one of the native ways of getting endianness.
E.g. see https://android.googlesource.com/platform/development/+/73a5a3b/ndk/platforms/android-20/include/sys/endian.h 
It seems you can do a check such as 
#if _BYTE_ORDER == _LITTLE_ENDIAN

See also https://stackoverflow.com/a/9189572/8528814

Reply:
Changed. PTAL.